performance-tests/jbpm-performance-tests/src/main/java/org/kie/perf/jbpm/constant/ProcessStorage.java

```diff
@@ -11,7 +11,9 @@
     ScriptTask("ScriptTask.bpmn2", "org.kie.perf.ScriptTask"),
     IntermediateSignal("IntermediateSignal.bpmn2", "org.kie.perf.IntermediateSignal"),
     StartEnd("StartEnd.bpmn2", "org.kie.perf.StartEnd"),
-    IntermediateTimer("IntermediateTimer.bpmn2", "org.kie.perf.IntermediateTimer")
+    IntermediateTimer("IntermediateTimer.bpmn2", "org.kie.perf.IntermediateTimer"),
+    AsyncPrintTask("AsyncPrintTask.bpmn2", "org.kie.perf.AsyncPrintTask"),
+    MultiInstanceSubprocesses("MultiInstanceSubprocesses.bpmn2", "org.kie.perf.MultiInstanceSubprocesses")
```

Comment:
The enum has two parameters - the BPMN file name and process ID, hence we would always have 2 static strings for 1 process definition. From the usage POV I see this easier for reuse.